pkg/core/blockchainer/state_root.go

```diff
@@ -2,12 +2,17 @@ package blockchainer
 
 import (
 	"github.com/nspcc-dev/neo-go/pkg/core/state"
+	"github.com/nspcc-dev/neo-go/pkg/crypto/keys"
 	"github.com/nspcc-dev/neo-go/pkg/util"
 )
 
 // StateRoot represents local state root module.
 type StateRoot interface {
+	AddValidatedStateRoot(root *state.MPTRoot) error
```

Comment:
Can we make it `AddStateRoot` that will do `VerifyStateRoot` internally? Exposing this is dangerous, it's easy to add broken state root via this interface.

Reply:
Fixed.